node/configs/src/chain_spec.rs

```diff
@@ -78,6 +78,7 @@ impl ChainSpec {
     /// Returns:
     /// * generated `ChainSpec`;
     /// * signers that can be used for assertions and mocking in tests.
+    #[allow(clippy::needless_range_loop)]
```

Comment:
`needless_range_loop` seems to be buggy in general, just disable it completely. https://github.com/rust-lang/rust-clippy/issues/398